styling-and-themes/overview.md

```diff
@@ -145,6 +149,22 @@ Here is the result with the `Purple` and `Purple Dark` swatches applied:
 
 ![Telerik .NET MAUI Theming App Usage](images/telerik-theming-app.png)
 
+## Changing the Initially Loaded Themes and Control's XAML Files
+
+You can decide which themes and control's XAML files to be generated in the `TelerikTheming` folder by modifying the `config.json` file.
+
+Once the app is build when `UseTelerikTheming` is set to `True`
+
+```XAML
```

Comment:
Don't repeat the same snippet, just link to the section where it is already provided.
```suggestion
```